routers/api/v1/activitypub/person.go

```diff
@@ -0,0 +1,121 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package activitypub
+
+import (
+	"net/http"
+	"strings"
+
+	"code.gitea.io/gitea/modules/activitypub"
+	"code.gitea.io/gitea/modules/context"
+	"code.gitea.io/gitea/modules/json"
+	"code.gitea.io/gitea/modules/log"
+	"code.gitea.io/gitea/modules/setting"
+
+	ap "github.com/go-ap/activitypub"
+)
+
+// Person function returns the Person actor for a user
+func Person(ctx *context.APIContext) {
+	// swagger:operation GET /activitypub/user/{username} activitypub activitypubPerson
+	// ---
+	// summary: Returns the Person actor for a user
+	// produces:
+	// - application/json
+	// parameters:
+	// - name: username
+	//   in: path
+	//   description: username of the user
+	//   type: string
+	//   required: true
+	// responses:
+	//   "200":
+	//     "$ref": "#/responses/ActivityPub"
+
+	link := strings.TrimSuffix(setting.AppURL, "/") + "/api/v1/activitypub/user/" + ctx.ContextUser.Name
+	person := ap.PersonNew(ap.IRI(link))
+
+	person.Name = ap.NaturalLanguageValuesNew()
```

Comment:
```suggestion
	person := &ap.Person{
		ID: ap.IRI(link),
		Type: ap.PersonType,
		Name: ap.NaturalLanguageValuesNew(),
		PreferredUsername: ap.NaturalLanguageValuesNew(),
	}

```

Reply:
This fixes the empty `liked` collection.

Reply:
More recent versions of go-ap/activitypub don't have this bug.